pyscraper/regmem/__main__.py

```diff
@@ -14,11 +14,17 @@ def cli():
 @click.option("--chamber", type=str, default="commons")
 @click.option("--force-refresh", is_flag=True)
 @click.option("--quiet", is_flag=True)
+@click.option("--no-progress", is_flag=True)
 def download_all_registers(
-    chamber: str, force_refresh: bool = False, quiet: bool = False
+    chamber: str,
+    force_refresh: bool = False,
+    quiet: bool = False,
+    no_progress: bool = False,
```

Comment:
I realise this may not be possible due to workflow, but in general it'd be great if formatting changes weren't in the same commit as logical changes, because it makes it much harder to tell what has actually changed and what is just noise.